thirdparty/kpvcrlib/CMakeLists.txt

```diff
@@ -200,4 +201,8 @@ add_library(crengine SHARED ${CRENGINE_SOURCES})
 # Make sure we'll get C++11 support
 target_compile_features(crengine PRIVATE cxx_constexpr)
 
-target_link_libraries(crengine chmlib antiword ${THIRDPARTY_LIBS})
+target_link_libraries(crengine PRIVATE chmlib antiword)
+target_link_libraries(crengine PUBLIC ${THIRDPARTY_LIBS})
```

Comment:
Should these thirdparty libs be PUBLIC ?
I thought earler that by using PRIVATE, cre.cpp linking to libcrengine would not know that libcrengine would be linked to use liblunasvg and would then get less change to be linked against libstdc++ - but well, it didn't work.

Reply:
As usual, CMake's doc is terrible, so this mail looks like a much, much, much better description: https://cmake.org/pipermail/cmake/2016-May/063400.html ;).

Reply:
TL;DR: I guess some of 'em could be private, but I'm way too lazy to sort it out, and the point is moslty moot (on sane binutils versions) as `--as-needed` will avoid overlinking stuff that pulls in libcrengine.so anyway.